test/data/adblock-data/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Pls ensure that the keys here are for a different public key than we use in prod. You might need a hack similar to `SetAdBlockURLForTest` for the tests.